### DISCUSSION TITLE
What is the default kernel and parameters for the simulator?

### DISCUSSION THREAD

**Discussion by xherdan76** [InteractiveComputerGraphics/SPlisHSPlasH#215] [Q&A]
Hello, thanks for the interesting work!

I noticed that the kernel for SPH is not specified in all the example scene files. I would like to know what is the default kernel used by the simulator and its parameters if no kernel is specified in the secene file.

Thank you.

**Comment by janbender** [comment]
This depends on the preferred kernel of the pressure solver. We took the kernel that was specified in the corresponding publication. You can see the setting in the code: 

https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/blob/master/SPlisHSPlasH/Simulation.cpp#L527
